lib/jsdom/browser/BrowsingContext.js

```diff
@@ -0,0 +1,58 @@
+"use strict";
+
+const idlUtils = require("../living/generated/utils");
+
+// https://html.spec.whatwg.org/#windows
+class BrowsingContext {
+  // readonly windowProxy: WindowProxy
+  // readonly sessionHistory: SessionHistory
+
+  // create a new browsing context
+  constructor() {
+    this.windowProxy = new WindowProxy();
+    this.sessionHistory = new SessionHistory();
+
+    // 2. Call the JavaScript InitializeHostDefinedRealm() abstract operation
+    const window = new Window();
+    // TODO: InitializeHostDefinedRealm({globalObject: window, globalThisValue: this.windowProxy, relmExecutionContext: new JavaScriptExecutionContext})
```

Comment:
I don't think this TODO is very actionable.

Creating a new realm is basically the vm.createContext() business, which the Window constructor takes care of. I'd maybe revise this to

```
// 2. Call ...
// In the current implementation, the Window() constructor takes care of creating a new realm via the vm module
```